staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -468,6 +468,10 @@ type CreateOptions struct {
 	// DryRun decides if the request should be completed or not.
 	// +optional
 	DryRun bool `json:"dryRun,omitempty" protobuf:"varint,1,opt,name=dryRun"`
+
+	// If IncludeUninitialized is specified, the object may be
+	// returned without completing initialization.
+	IncludeUninitialized bool `json:"includeUninitialized,omitempty" protobuf:"varint,2,opt,name=includeUninitialized"`
```

Comment:
this surprises me. How is this related to dry-run? Why do we expose that via an API "suddenly"?

Reply:
Also note that initializers go away (if I have understood @smarterclayton correctly). We should not expose a parameter for something that is planned to be removed.

Reply:
Ok, found it: https://github.com/kubernetes/kubernetes/pull/65105/commits/69e757ae6077d31376512e8b6b3e54fbe90a30f6#diff-7861f33474e4eb8d21cf1d33de5e4d4eL114 had the logic before, with a manually parsed get var. So nvm my comment above.